src/uiTest/scala/com/virtuslab/gitmachete/uitest/UIScenarioSuite.scala

```diff
@@ -0,0 +1,103 @@
+package com.virtuslab.gitmachete.uitest
+
+import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite
+import com.virtuslab.gitmachete.testcommon.SetupScripts.SETUP_README_SCENARIOS
+import org.junit._
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.virtuslab.ideprobe.ProbeDriver
+
+@RunWith(classOf[JUnit4])
+class UIScenarioSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_README_SCENARIOS) {
+
+  import UISuite._
+  UISuite.setup()
+
+  private val project = intelliJ.project
+
+  private val probe: ProbeDriver = intelliJ.probe
+
+  @Before
+  def beforeEach(): Unit = {
+    probe.openProject(rootDirectoryPath)
+    project.configure()
+    probe.await()
+    project.openGitMacheteTab()
+    project.toggleListingCommits()
+    intelliJ.ide.findAndResizeIdeFrame()
+  }
+
+  @After
+  def afterEach(): Unit = {
+    probe.await()
+    // Note that we shouldn't wait for a response here (so we shouldn't use org.virtuslab.ideprobe.ProbeDriver#closeProject),
+    // since the response sometimes never comes (due to the project being closed), depending on the specific timing.
+    intelliJ.ide.closeOpenedProjects()
+  }
+
+  @Ignore
```

Comment:
Hmmm so for a local run, `@Ignore` would need to be manually removed from here? 🤔

Reply:
yes

Reply:
Actually... maybe `@Ignore` on the entire class, if possible? will be more visible/clear

Reply:
Plus add a comment why it's ignored